Service/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,36 @@
+package org.example;
+
+import TIcket.Ticket;
+import Users.Admin;
+import Users.User;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Main extends SetClassId implements Print {
```

Comment:
Looks like we don't actually need to extends here and implement, extends SetClassId implements Print is useless here, we don't use them